### ISSUE TITLE
can not access `https://dgraph.io/`

### ISSUE THREAD

**Issue by goldwind-ting** [dgraph-io/dgraph#8140]
**GitHub Issues are deprecated. Use [Discuss Issues](https://discuss.dgraph.io/c/issues/dgraph/38) for reporting issues about this repository.**

**Comment by skrdgraph** [comment]
@goldwind thank you for reporting the issue. This issue was resolved today at 15:17 UTC. You can also track our status page [here](https://status.dgraph.io).

PS: We don't track issues on `GitHub`. Our support team uses [discuss](https://discuss.dgraph.io/c/issues/dgraph). 

fyi @rarvikar

**Comment by goldwind-ting** [comment]
@skrdgraph still can not access `https://dgraph.io/`.

**Comment by skrdgraph** [comment]
@goldwind-ting - we are discussing this internally atm & will have a permanent fix (hopefully soon). Just to clarify what you experienced - there were intermittent 5xx issues we noticed on our end as well. 

The website is available now and running fine. We can `re-open` this issue if this persists & repeats - and I or @rarvikar will circle back here once we have addressed this with a permanent fix to close the loop.
